src/paperless_mail/tests/test_parsers_live.py

```diff
@@ -0,0 +1,288 @@
+import hashlib
+import os
+from unittest import mock
+from urllib.error import HTTPError
+from urllib.request import urlopen
+
+import pytest
+from django.test import TestCase
+from documents.parsers import ParseError
+from documents.parsers import run_convert
+from paperless_mail.parsers import MailDocumentParser
+from pdfminer.high_level import extract_text
+
+
+class TestParserLive(TestCase):
+    SAMPLE_FILES = os.path.join(os.path.dirname(__file__), "samples")
+
+    def setUp(self) -> None:
+        self.parser = MailDocumentParser(logging_group=None)
+
+    def tearDown(self) -> None:
+        self.parser.cleanup()
+
+    @staticmethod
+    def hashfile(file):
```

Comment:
This is true for the sample files I'm reading, but since it is a generic function which could be used for large files, I'd rather have the limit.
buf_size is already 10x bigger than the biggest file I'm hashing, so it should be good.

Reply:
outdated